static/js/tools/stat_var/page.tsx

```diff
@@ -162,13 +207,13 @@ class Page extends Component<unknown, PageStateType> {
         : undefined;
     this.setState({
       description:
-        descriptionPromise.data[sv].length > 0
-          ? descriptionPromise.data[sv][0]
+        descriptionPromise?.data[sv].length > 0
```

Comment:
sorry, one more thing, we should probably also handle failed promises here

Reply:
done